### ISSUE TITLE
(tlg0655.tlg001) Epidoc and CTS conversion 

### ISSUE THREAD

**Issue by jdeen33** [PerseusDL/canonical-greekLit#503]
claiming file for conversion work

**Comment by jdeen33** [comment]
@lcerrato CHS intern here! please bump the URN for file tlg0655.tlg001

**Comment by lcerrato** [comment]
Hi, @jdeen33 
tlg0655.tlg001.perseus-grc1.xml, correct?

**Comment by jdeen33** [comment]
Hi @lcerrato yes that's correct, thank you!

**Comment by lcerrato** [comment]
ok, CTS URN tlg0655.tlg001.perseus-grc2 has been created.

**Comment by jdeen33** [comment]
@lcerrato the CTS structure in the original file is chapter/section --would you recommend keeping that?

**Comment by lcerrato** [comment]
@jdeen33 
Yes, that should be fine. Looks like you will need two unnumbered sections prior to chapter 1: argument and preface.

**Comment by jdeen33** [comment]
fix #522
